### PR TITLE
Typo "Azure File Storage"→"Azure Files Storage"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
## Summary
Typo "Azure File Storage"→"Azure Files Storage"

https://learn.microsoft.com/en-us/dotnet/fsharp/using-fsharp-on-azure/file-storage
https://github.com/dotnet/docs/blob/main/docs/fsharp/using-fsharp-on-azure/file-storage.md
#PingMSFTDocs

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by gewarren** [comment]
It seems like "Azure Files" is capitalized but "storage" is not part of the official name and should be lower case? https://learn.microsoft.com/en-us/azure/cloud-shell/quickstart#next-steps

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
-title: Get started with Azure File Storage using F#
-description: Store file data in the cloud with Azure File Storage, and mount your cloud file share from an Azure virtual machine (VM) or from an on-premises application running Windows.
+title: Get started with Azure Files Storage using F#
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:2]:
```suggestion
title: Get started with Azure Files using F#
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+description: Store file data in the cloud with Azure Files Storage, and mount your cloud file share from an Azure virtual machine (VM) or from an on-premises application running Windows.
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:3]:
```suggestion
description: Store file data in the cloud with Azure Files and mount your cloud file share from an Azure virtual machine (VM) or from an on-premises application running Windows.
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 author: sylvanc
 ms.date: 08/26/2021
 ms.custom: "devx-track-fsharp"
 ---
-# Get started with Azure File Storage using F\#
+# Get started with Azure Files Storage using F\#
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:8]:
```suggestion
# Get started with Azure Files using F\#
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+Azure Files Storage is a service that offers file shares in the cloud using the standard [Server Message Block (SMB) Protocol](/windows/win32/fileio/microsoft-smb-protocol-and-cifs-protocol-overview). Both SMB 2.1 and SMB 3.0 are supported. With Azure Files Storage, you can migrate legacy applications that rely on file shares to Azure quickly and without costly rewrites. Applications running in Azure virtual machines or cloud services or from on-premises clients can mount a file share in the cloud, just as a desktop application mounts a typical SMB share. Any number of application components can then mount and access the File storage share simultaneously.
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:10]:
```suggestion
Azure Files is a service that offers file shares in the cloud using the standard [Server Message Block (SMB) protocol](/windows/win32/fileio/microsoft-smb-protocol-and-cifs-protocol-overview). Both SMB 2.1 and SMB 3.0 are supported. With Azure Files, you can migrate legacy applications that rely on file shares to Azure quickly and without costly rewrites. Applications running in Azure virtual machines or cloud services or from on-premises clients can mount a file share in the cloud, just as a desktop application mounts a typical SMB share. Any number of application components can then mount and access the file storage share simultaneously.
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 
-For a conceptual overview of file storage, see [the .NET guide for file storage](/azure/storage/storage-dotnet-how-to-use-files).
+For a conceptual overview of File Storage, see [the .NET guide for File Storage](/azure/storage/storage-dotnet-how-to-use-files).
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:12]:
```suggestion
For a conceptual overview of file storage, see [the .NET guide for file storage](/azure/storage/storage-dotnet-how-to-use-files).
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 6]
```diff
@@ -133,12 +133,12 @@ You can enable metrics for File storage from the [Azure portal](https://portal.a
 
 ## Next steps
 
-For more information about Azure File Storage, see these links.
+For more information about Azure Files Storage, see these links.
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:136]:
```suggestion
For more information about Azure Files, see these links.
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 7]
```diff
@@ -133,12 +133,12 @@ You can enable metrics for File storage from the [Azure portal](https://portal.a
...
 
 ### Conceptual articles and videos
 
 - [Azure Files Storage: a frictionless cloud SMB file system for Windows and Linux](https://azure.microsoft.com/resources/videos/azurecon-2015-azure-files-storage-a-frictionless-cloud-smb-file-system-for-windows-and-linux/)
-- [How to use Azure File Storage with Linux](/azure/storage/storage-how-to-use-files-linux)
+- [How to use Azure Files Storage with Linux](/azure/storage/storage-how-to-use-files-linux)
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:141]:
```suggestion
- [How to use Azure Files with Linux](/azure/storage/storage-how-to-use-files-linux)
```

File: docs/fsharp/using-fsharp-on-azure/file-storage.md [hunk 8]
```diff
@@ -153,7 +153,7 @@ For more information about Azure File Storage, see these links.
 
 ### Blog posts
 
-- [Azure File Storage is now generally available](https://azure.microsoft.com/blog/azure-file-storage-now-generally-available/)
-- [Inside Azure File Storage](https://azure.microsoft.com/blog/inside-azure-file-storage/)
+- [Azure Files Storage is now generally available](https://azure.microsoft.com/blog/azure-file-storage-now-generally-available/)
```
Comment (gewarren) [inline docs/fsharp/using-fsharp-on-azure/file-storage.md:156]:
```suggestion
- [Azure Files storage is now generally available](https://azure.microsoft.com/blog/azure-file-storage-now-generally-available/)
```

**Review by gewarren — COMMENTED** [review]
It looks to me like the name has actually evolved to drop the "storage" part since 2015, so I left some suggestions. Thanks!

**Comment by hyoshioka0128** [comment]
@gewarren - Thanks!

**Review by gewarren — APPROVED** [review]

**Comment by gewarren** [comment]
Thanks @hyoshioka0128

**Pull request merged**